doc/man1/flux-shell.rst

```diff
@@ -0,0 +1,446 @@
+==============
+flux-shell(1)
+==============
+
+
+SYNOPSIS
+========
+
+**flux-shell** [*OPTIONS*] *JOBID*
+
+DESCRIPTION
+===========
+
+flux-shell(1), the Flux job shell, is the component of Flux which manages
+the startup and execution of user jobs. flux-shell(1) runs as the job user,
+reads the jobspec and assigned resource set R for the job from the KVS,
+and using this data determines what local job tasks to execute. While job
+tasks are running, the job shell acts as the interface between the parent
+instance and the job by handling I/O, signals, and finally collecting
```

Comment:
"Parent instance" might be slightly confusing here since we're not talking about nested instances (necessarily).  Would it be better to just say "Flux" or "Flux instance"?

Reply:
Good catch!